gui/src/euRun.cc

```diff
@@ -174,7 +174,54 @@ void RunControlGUI::OnReceive(const eudaq::ConnectionInfo &id,
     EmitStatus("EVENT", status->GetTag("EVENT"));
     EmitStatus("FILEBYTES", to_bytes(status->GetTag("FILEBYTES")));
   } else if (id.GetType() == "Producer") {
-    if (id.GetName() == "TLU" || id.GetName() == "miniTLU") {
+ if (id.GetName() == "caliceahcalbif") {
+      EmitStatus("TRIG", status->GetTag("TRIG"));
+      EmitStatus("PARTICLES", status->GetTag("PARTICLES"));
+      // EmitStatus("TIMESTAMP", status->GetTag("TIMESTAMP"));
```

Comment:
I  think it is better not to have "dead" / out commented code in the repository. Probably you could remove it? May I ask why you need the special case if you are running with your producer? As far as I see it you are just not showing some entries. You are running without a TLU?